Gemfile

```diff
@@ -2,25 +2,22 @@
 
 source 'https://rubygems.org'
 
+# stanford dlss specific gems
 gem 'assembly-objectfile'
-gem 'config', '~> 3.1'
 gem 'dor-services-client', '~> 12.0'
 gem 'dor-workflow-client', '~> 4.0'
 gem 'druid-tools'
-gem 'fastimage', '~> 2.2'
-gem 'ffi-geos', '~> 1.0', require: false # XXX: where is this used?
-# iso-639 0.3.0 isn't compatible with ruby 2.5.  This declaration can be dropped when we upgrade to ruby 2.6
-# see https://github.com/alphabetum/iso-639/issues/12
-gem 'geoserver-publish', '>= 0.5.0'
-gem 'iso-639', '~> 0.2.10'
```

Comment:
the comment refers to the iso-639 gem inclusion, and we are well past ruby 2.6 now